fdbserver/storageserver.actor.cpp

```diff
@@ -341,12 +341,19 @@ struct StorageServer {
 			UID UIDofLongest;
 			for (const auto& kv: startTimeMap) {
 				const double currentRunningTime = currentTime - kv.second;
-				if (longest < currentRunningTime) {
+				if (longest <= currentRunningTime) {
 					longest = currentRunningTime;
 					UIDofLongest = kv.first;
 				}
 			}
-			return {longest, keyRangeMap.at(UIDofLongest)};
+			if(BUGGIFY) {
+				UIDofLongest = deterministicRandom()->randomUniqueID();
+			}
+			if(keyRangeMap.count(UIDofLongest)) {
+				const KeyRange& range = keyRangeMap[UIDofLongest];
```

Comment:
The problem is that `std::unordered_map::operator[]` is not const because it can modify the map by adding a default value if the key is not present, and you can only do const operations on keyRangeMap since we're in a const method (a.k.a `this` is const)